### DISCUSSION TITLE
Open beatmaps while logged out

### DISCUSSION THREAD

**Discussion by mmdevcodes** [ppy/osu#17517] [Q&A]
Currently you cannot open any beatmap pages ingame or via the browser because both are locked behind being logged in.

Personally I think the best user experience would allow beatmap pages to be public regardless of logged in status. At the very least though you should be able to open them externally as a separate button somewhere.

**Comment by peppy** [comment]
Very low priority, but I'd expect it to work the same as the site at the end of the day (some views should work even as a guest).

**Reply by mmdevcodes** [reply]
For the time being can we please have the browser just open up the page if you're logged out? Not being able to open any beatmaps is pretty huge.

**Reply by peppy** [reply]
Please login instead.

**Reply by mmdevcodes** [reply]
Last week when I asked about a feature for [appearing offline ](https://github.com/ppy/osu/discussions/17440) the answer was to just logout. Now I'm logged out and the answer is to just login. Cmon peppy throw me a bone here please.

**Reply by peppy** [reply]
Here's a bone:

Either login and use our servers, access beatmaps etc.

Or don't login and figure things out yourself.

When 99.999999% of users are happy to login, and we *require* login for many reasons, I cannot give you my dedicated support just because you have a special use case.
